index.bs

```diff
@@ -1927,6 +1927,12 @@ through the properties of the {{IDBRequest}} instance.
 
 The <span>task source</span> for these tasks is the <dfn id=database-access-task-source export>database access task source</dfn>.
 
+<div algorithm>
+
+To <dfn export>queue a database task</dfn>, perform [=queue a task=] on the [=database access task source=]
```

Comment:
Nit: needs trailing period at the end of the sentence.

Why export the definition? Are you aware of other specs that should be referencing it?